src/references.md

```diff
@@ -17,161 +17,140 @@ Which obey the following rules:
 * A reference cannot outlive its referent
 * A mutable reference cannot be aliased
 
-That's it. That's the whole model. Of course, we should probably define
-what *aliased* means. To define aliasing, we must define the notion of
-*paths* and *liveness*.
+That's it. That's the whole model references follow.
 
+Of course, we should probably define what *aliased* means. An aliased--
 
-**NOTE: The model that follows is generally agreed to be dubious and have
-issues. It's ok-ish as an intuitive model, but fails to capture the desired
-semantics. We leave this here to be able to use notions introduced here in later
-sections. This will be significantly changed in the future. TODO: do that.**
+```text
+error[E0425]: cannot find value `aliased` in this scope
+ --> <rust.rs>:2:20
+  |
+2 |     println!("{}", aliased);
+  |                    ^^^^^^^ not found in this scope
 
+error: aborting due to previous error
+```
 
-# Paths
+Unfortunately, Rust hasn't actually defined its aliasing model. 🙀
 
-If all Rust had were values (no pointers), then every value would be uniquely
-owned by a variable or composite structure. From this we naturally derive a
-*tree* of ownership. The stack itself is the root of the tree, with every
-variable as its direct children. Each variable's direct children would be their
-fields (if any), and so on.
+While we wait for the Rust devs to specify the semantics of their language,
+let's discuss what aliasing is in general, and why it matters.
 
-From this view, every value in Rust has a unique *path* in the tree of
-ownership. Of particular interest are *ancestors* and *descendants*: if `x` owns
-`y`, then `x` is an ancestor of `y`, and `y` is a descendant of `x`. Note
-that this is an inclusive relationship: `x` is a descendant and ancestor of
-itself.
 
-We can then define references as simply *names* for paths. When you create a
-reference, you're declaring that an ownership path exists to this address
-of memory.
 
-Tragically, plenty of data doesn't reside on the stack, and we must also
-accommodate this. Globals and thread-locals are simple enough to model as
-residing at the bottom of the stack (though we must be careful with mutable
-globals). Data on the heap poses a different problem.
 
-If all Rust had on the heap was data uniquely owned by a pointer on the stack,
-then we could just treat such a pointer as a struct that owns the value on the
-heap. Box, Vec, String, and HashMap, are examples of types which uniquely
-own data on the heap.
+# Aliasing
 
-Unfortunately, data on the heap is not *always* uniquely owned. Rc for instance
-introduces a notion of *shared* ownership. Shared ownership of a value means
-there is no unique path to it. A value with no unique path limits what we can do
-with it.
+Variables and pointers are generally said to *alias* if they refer to the same
+location in memory.
```

Comment:
I can see that. However, in that case, there is no "single location" in memory that they point to, which the wording in the text somewhat implies -- "refer to *the* same location" makes it sound like "the location a pointer points to" is a well-defined concept, whereas really it is a set of locations.